src/sentry/static/sentry/app/views/eventsV2/events.tsx

```diff
@@ -60,11 +61,16 @@ export default class Events extends React.Component<EventsProps> {
             fixed: 'events chart',
           })}
         </Panel>
-        <StyledSearchBar
+        <Feature
           organization={organization}
-          query={query}
-          onSearch={this.handleSearch}
-        />
+          features={['events-v2', 'discover-v2-query-builder']}
+        >
+          <StyledSearchBar
```

Comment:
We'll want to continue showing this search bar when the user doesn't have `discover-v2-query-builder`. When the org does have this feature we'll want to show more/different UI controls inside the `<Table>` component.